docs/sql-ref-syntax-ddl-create-table-datasource.md

```diff
@@ -40,7 +40,7 @@ CREATE TABLE [ IF NOT EXISTS ] table_identifier
     [ AS select_statement ]
 ```
 
-Note that, the clauses between the USING clause and the AS SELECT clause can come in
+Note that, the clauses between the OPTIONS clause and the AS SELECT clause can come in
```

Comment:
Should still be `USING` since `OPTIONS` itself can also come in any order?

Reply:
between is more likely be inclusive, otherwise previous statement was also wrong.

Reply:
> between the USING clause and the AS SELECT clause

USING and AS SELECT are both not included, so this is still corrected.

Reply:
Ah OK my bad. I found USING shouldn't com in any order so previous statement was correct. Sorry about that.